src/commands/bundle.zsh

```diff
@@ -9,7 +9,7 @@ antigen-bundle () {
     return 1
   fi
 
-  builtin typeset -A bundle; -antigen-parse-args 'bundle' ${=@}
+  builtin typeset -A bundle; -antigen-parse-args 'bundle' $@
```

Comment:
If I change this back to `${=@}` then it breaks the test I added because the spaces get turned into different arguments, unfortunately. Is there another way to parse args after they've been send to `-antigen-parse-args`?

I can also look into sending the arguments a different way if you like